src/komponenter/Banner/BannerNAVAnsatt.tsx

```diff
@@ -1,48 +1,76 @@
 import { InnloggetBrukerContext } from '@/InnloggingBoundary/InnloggingBoundary';
 import '@navikt/bedriftsmeny/lib/bedriftsmeny.css';
 import amplitude from 'amplitude-js';
-import { Heading, Detail } from '@navikt/ds-react';
+import { Heading, Detail, Alert } from '@navikt/ds-react';
 import React, { useContext } from 'react';
 import VerticalSpacer from '../layout/VerticalSpacer';
 import nyheter from '../NyttIAppen/nyheter';
 import Nytt from '../NyttIAppen/Nytt';
 import './Banner.less';
+import moment from 'moment/moment';
+import { useAvtale } from '@/AvtaleProvider';
+import { useFeatureToggles } from '@/FeatureToggleProvider';
 
 interface Props {
     tekst: string;
     undertittel?: string;
 }
 
+const formatertDato = (sistEndret: string) => {
+    const sistEndretPlussTolvUker = moment(sistEndret).startOf('day').add(12, 'weeks');
+    const dager = sistEndretPlussTolvUker.diff(moment(), 'days');
+    if (dager === 0) {
+        return `kl. 23:59 i dag`;
```

Comment:
Blir det riktrig da? alt under 1 dag blir 23:59?
Vi har forøvrig en egen fil: datoUtils.ts hvor denne funksjonen kanskje passer bedre. Her har vi også en formatterTid funksjon. Kanskje den kan gjenbrukes?

Reply:
Ja, alle som har mindre enn 1 dag vil plukkes opp av jobben kl 00:00. Så da vil veileder ha til 23:59 den dagen å foreta seg noe.